README.md

```diff
@@ -64,13 +64,15 @@ avroProducer.produce(topic='my_topic', value=value, key=key)
 ```
 from confluent_kafka.avro import AvroConsumer
 
-c = AvroConsumer({'bootstrap.servers': 'mybroker,mybroker2', 'group.id': 'greoupid', 'schema.registry.url': 'http://127.0.0.1:9002'})
+c = AvroConsumer({'bootstrap.servers': 'mybroker,mybroker2', 'group.id': 'groupid', 'schema.registry.url': 'http://127.0.0.1:8081'})
 c.subscribe(['my_topic'])
 running = True
 while running:
     msg = c.poll(10)
     if msg:
         if not msg.error():
+            if (msg.value() && isinstance(msg.value(), dict)):
```

Comment:
Is this ever not true?
From what I understand the deserializer will raise an exception if it can't deserialize a message.
Speaking of which, this example should probably have a proper try:..except block catching such errors, right?